slm_porosity/heat_transfer.f90

```diff
@@ -230,6 +234,8 @@ SUBROUTINE user_initial_conditions (u, nlocal, ne_local, t_local, scl, scl_fltwt
       !   b) update it by solving the corresponding equation
       func_newton => enthalpy_equation
       u(:,n_var_enthalpy) = find_root(u(:,n_var_enthalpy), temp)
+    ELSEIF ( IC_restart_mode.EQ.0 ) THEN
+      u(:,n_var_enthalpy) = 0.0_pr
```

Comment:
Do we need this flag? Why we can't use `initial_temperature=0`?

Reply:
I already told that there are errors with intial_temperature = 0 and even with = 0.02-0.8. If bug would be detected this flag can be deleted.